pom.xml

```diff
@@ -21,17 +21,45 @@
         <module>spigot-impl</module>
         <module>velo-impl</module>
         <module>discord-impl</module>
-        <module>spigot-examples</module>
-
+        <module>spigot-commons</module>
+        <module>paper-impl</module>
     </modules>
 
-    <properties>
-
-        <beetle.version>0.2.0</beetle.version>
-
-        <core.version>0.3.3</core.version>
-        <default.version>0.3.3</default.version>
+    <build>
+        <plugins>
+            <plugin>
+                <configuration>
```

Comment:
This configuration section is poorly placed. It should come after the groupId/artifactId/version trifecta